migrations/0035-add-handle-column-to-users-table.ts

```diff
@@ -0,0 +1,36 @@
+import { MigrationInterface, QueryRunner, TableColumn, TableIndex } from 'typeorm';
+
+const TABLE_NAME = 'Users';
+
+export class AddHandleColumnToUsersTable1667512340166 implements MigrationInterface {
+  public async up(queryRunner: QueryRunner): Promise<void> {
+    // Add "handle" column to "Users" table as nullable initially since it does not contain any data
+    await queryRunner.addColumn(TABLE_NAME, new TableColumn({
+      name: 'handle',
+      type: 'varchar(255)',
+      isNullable: true,
+      isUnique: true,
+    }));
+
+    await queryRunner.createIndex(TABLE_NAME, new TableIndex({
+      name: 'users_by_handle_index',
+      columnNames: ['handle'],
+      isUnique: true,
+    }));
+
+    // Populate the "handle" column of "Users" table in every row with a default value
+    // formatted as {firstName}{lastName}{hash}
+    await queryRunner.query(`
+      UPDATE "${TABLE_NAME}"
+      SET handle = "firstName" || '-' || "lastName" || '-' || substr(md5(random()::text), 0, 7)
+    `);
+
```

Comment:
nit: let's update this comment to add the dashes

Reply:
Also, what if this default value is longer than 32 characters? Should we apply our length constraints here as well (similar to how we do in `UserAccountService`?)